instrument/Facilities.xml

```diff
@@ -716,15 +709,15 @@
   </instrument>
 
   <instrument name="D2B">
-    <technique>Powder diffraction</technique>
+    <technique>PowderDiffraction</technique>
   </instrument>
 
   <instrument name="D20">
-    <technique>Powder diffraction</technique>
+    <technique>PowderDiffraction</technique>
   </instrument>
 
   <instrument name="D1B">
-    <technique>Powder diffraction</technique>
+    <technique>PowderDiffraction</technique>
   </instrument>
 
   <instrument name="D4C">
```

Comment:
Line below could be modified to `LiquidDiffraction` to follow the diffraction pattern above.

Reply:
That's a kind of diffraction pattern I never saw before !